corehq/apps/userreports/templates/userreports/summary_data_source.html

```diff
@@ -54,7 +54,7 @@ <h2>Named Expressions</h2>
     </tr>
 {% for expression in named_expression_summary %}
     <tr>
-        <td id="NamedE:{{ expression.name }}">{{ expression.name }}</td>
+        <td id="{{ named_expression_prefix }}:{{ expression.name }}">{{ expression.name }}</td>
```

Comment:
Couldn't this also just use `{{ expression }}` or something to use the implementation in `__str__`?  It's unfortunate to have this duplicated, because if `__str__` changes in the future, this link would break.

Reply:
actually I guess this is duplicated in the link generation thing anyways.  Hmm, that's tricky.  I suppose you could do something else for link so the string display of the expression and the HTML id are different things.  This seems fine, no strong opinions.

Reply:
Cool - I guess we could generate the link in the View and pass that through to use here if we want it less brittle.